CONTRIBUTING.md

```diff
@@ -61,3 +61,11 @@ mypy --config-file=mypy.ini -p pytket -p tests
 We use [pylint](https://pypi.org/project/pylint/) on the CI to check compliance
 with a set of style requirements (listed in `pytket/.pylintrc`). You should run
 `pylint` over any changed files before submitting a PR, to catch any issues.
+
+## Version numbers
+
+If you make any changes in `thet/src`, you should bump the version number of
```

Comment:
Is there a way to make this automated on the CI?

Reply:
Are there defined changes that required which changes in the "part" of the version?

Reply:
> Is there a way to make this automated on the CI?

Could be done with some scriptery, but might be more trouble than it's worth. Autogenerated conanfiles scare me a little.

Reply:
> Are there defined changes that required which changes in the "part" of the version?

Good question. I think it would be good to follow semver here. I'll add a note abut this to the README.

Reply:
> > Is there a way to make this automated on the CI?
> 
> Could be done with some scriptery, but might be more trouble than it's worth. Autogenerated conanfiles scare me a little.

Okay, yes I can understand that

Reply:
Are there any bad side effects expected if the version is not updated in a first CI run before a review?

Reply:
> Are there any bad side effects expected if the version is not updated in a first CI run before a review?

I hope not. Packages only get uploaded to the artifactory on push to develop.